Public/Src/Engine/Processes/SandboxedProcessInfo.cs

```diff
@@ -99,7 +89,8 @@ public sealed class SandboxedProcessInfo
             bool testRetries = false,
             LoggingContext loggingContext = null,
             IDetoursEventListener detoursEventListener = null,
-            ISandboxConnection sandboxConnection = null)
+            ISandboxConnection sandboxConnection = null,
+            SharedOpaqueJournal sharedOpaqueOutputsJournal = null)

```

Comment:
>SharedOpaqueJournal sharedOpaqueOutputsJournal  [](start = 12, length = 47)

This has to be made serializable to be able to run in VM.

Reply:
I saw that but I was kind of hoping this wouldn't be necessary.

I'll make it serializable

Reply:
implemented serialization